### PR TITLE
Expected results

### PR DESCRIPTION
**Pull request by Jeffrey-Simpson**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # 1231

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Scripts at https://github.com/GRIDAPPSD/gridappsd-testing
Run with test_1.py and test_2.py  or 
request_test_expected_vs_running.py 
request_test_running_vs_timeseries.py
request_test_expected_vs_timeseries.py 
request_test_timeseries_vs_timeseries.py

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ X ] My code follows the style guidelines of this project
- [ X ] I have performed a self-review of my own code
- [ X ] I have commented my code, particularly in hard-to-understand areas
- [ X ] I have made corresponding changes to the documentation
- [ X ] My changes generate no new warnings
- [ X ] I have added tests that prove my fix is effective or that my feature works
- [ X ] New and existing unit tests pass locally with my changes
- [ X ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by poorva1209** [comment]
@Jeffrey-Simpson I ran test_1.py and test_2.py.  I can see the test output on both gridappsd terminal and gridappsd-testing terminal. 
I don't see any errors but I am not sure what I am seeing is expected and correct. Can we do a quick call and you can confirm my tests?

**Comment by Jeffrey-Simpson** [comment]
Hello,
Yes we can have a call.
Thanks,
-Jeff

File: services/fncsgossbridge/service/fncs_goss_bridge.py [hunk 1]
```diff
@@ -374,6 +374,14 @@ def run_simulation(self,run_realtime, archive_db_file, archive_file, only_archiv
...
         message = {}
         try:
+            headers_dict = yaml.safe_load(str(headers))
+            destination = headers_dict['destination']
+            message_id = headers_dict['message-id']
+            if str(destination).startswith('/temp-queue'):
+                return
+            if str(message_id) in self.message_id_list:
+                return
+            self.message_id_list.append(str(message_id))
```
[the remark this answers is not in the document]
Comment (Jeffrey-Simpson) [reply services/fncsgossbridge/service/fncs_goss_bridge.py]: Nope. I undid the commit and merged with the latest from develop.

File: services/fncsgossbridge/service/fncs_goss_bridge.py [hunk 2]
```diff
@@ -1209,7 +1217,7 @@ def _create_cim_object_map(map_file=None):
                             property_name = "voltage_" + phases;
                         elif measurement_type == "Pos":
                             object_name = conducting_equipment_name
-                            property_name = "status"
+                            property_name = "phase_" + phases + "_state"
```
[the remark this answers is not in the document]
Comment (Jeffrey-Simpson) [reply services/fncsgossbridge/service/fncs_goss_bridge.py]: Nope. I undid the commit and merged with the latest from develop.

**Pull request merged**